calibration/CTDMOG/CGINS-CTDMOG-13590__20170411.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13590,CC_ptempa0,1.547759e+002,
 37-13590,CC_ptempa1,-6.707572e-002,
 37-13590,CC_ptempa2,2.710008e-007,
-37-13590,CC_wbotc,4.0017e-007,
+37-13590,CC_wbotc,4.001652e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT